bin/compile

```diff
@@ -1,22 +1,45 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
-indent() {
-  sed -u 's/^/       /'
-}
+# fail fast
+set -e
+
+# debug
+# set -x
+
+# parse and derive params
 
-echo "-----> Installing tippecanoe"
 BUILD_DIR=$1
+CACHE_DIR=$2
+
+function indent() {
+  c='s/^/       /'
+  case $(uname) in
+    Darwin) sed -l "$c";;
+    *)      sed -u "$c";;
+  esac
+}
+
+function error() {
+  echo " !     $*" >&2
+  exit 1
+}
+
+function topic() {
+  echo "-----> $*"
+}
+
+topic "Downloading tippecanoe"
+
+tippecanoe_dir="$BUILD_DIR/tippecanoe"
+mkdir -p $tippecanoe_dir
 
 dep_url=https://github.com/mapbox/tippecanoe.git
-tippecanoe_dir=tippecanoe
-git clone $dep_url -q
-if [ ! -d "$tippecanoe_dir" ]; then
-  echo "[ERROR] Failed to find tippecanoe directory $tippecanoe_dir"
-  exit
-fi
+git clone $dep_url $tippecanoe_dir -q | ident
```

Comment:
please pin the version of tippecanoe we use here.

Reply:
✅